MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,194 @@
 import UIKit
 
 final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    //MARK: - Outlets
+
+    @IBOutlet weak var movieImage: UIImageView!
+    @IBOutlet weak var movieQuestion: UILabel!
+    @IBOutlet weak var movieCount: UILabel!
+    
+    // MARK: - Variables
+    private var currentQuestionIndex: Int = 0
+    private var currentQuestion: QuizQuestion {
+        questions[currentQuestionIndex]
+    }
+    private var correctAnswers: Int = 0
+    
+    
+    // MARK: - Actions
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        let givenAnswer = false
+        showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+    
+        
+    }
```

Comment:
Лишние пустые строки принято убирать, а между логическими блоками оставляют только одну строку
```suggestion
    }
    
```